src/vivarium_cluster_tools/vipin/perf_report.py

```diff
@@ -193,71 +195,84 @@ def print_stat_report(perf_df: pd.DataFrame, scenario_cols: list):
 
 def append_perf_data_to_central_logs(perf_df: pd.DataFrame, output_directory: Path) -> None:
     """Append performance data to the central logs."""
-    perf_df = perf_df.reset_index()
-    # add location data to perf_df
+    NUM_ROWS_PER_FILE = 100_000
+
+    central_perf_df = perf_df.reset_index().copy()
+    # add location data to central_perf_df
     artifact_path_col = "scenario_input_data_artifact_path"
-    if artifact_path_col in perf_df.columns:  # if we parallelized across artifact paths
-        perf_df["location"] = perf_df[artifact_path_col].apply(
+    if (
+        artifact_path_col in central_perf_df.columns
+    ):  # if we parallelized across artifact paths
+        central_perf_df["location"] = central_perf_df[artifact_path_col].apply(
             lambda filepath: Path(filepath).stem
         )
     else:  # else get from output directory
-        perf_df["location"] = output_directory.parents[3].stem
+        central_perf_df["location"] = output_directory.parents[3].stem
```

Comment:
Oh nice...I didn't know you could pass in the number of parents. I always went `xxx.parent.parent.parent.parent...`

Reply:
I actually learned about parents from reading one of your PRs haha